src/app.js

```diff
@@ -4,14 +4,27 @@ const negate = require('lodash/negate');
 const pickBy = require('lodash/pickBy');
 const project = require('../package.json');
 
+const buildEnvironment = (environment) => {
+  if (!environment) {
+    return null;
+  }
+
+  const environments = [];
+  for (const [ name, value ] of Object.entries(environment)) {
```

Comment:
This loop is already pretty simple, but lodash has a function that will do 50% of this work for you. I think this could be just

```js
const environments = _.map(environment, (name, key) => {{mapped_to: name, value}});
```

Reply:
Nice! I need a linter that can tell me these things as my brain isn't big enough to keep up with all of lodash 😉 I'll try to remember to tidy this up in a future PR.